### PR TITLE
Change version to 2.2-1 and update changelogs

### PR DESCRIPTION
**Pull request by pakmarkthub**
This PR:
- is for issue #169.
- bumps up version to 2.2. 
- resets package version to 1.
- updates copyright year.
- adds copyright text to all Makefiles, config_arch, build_*_packages.sh.
- updates changelogs.
- updates README to use lower-case `prefix` instead of the upper-case one.

### CODE REVIEW

File: packages/debian/changelog [hunk 1]
```diff
@@ -1,5 +1,21 @@
 gdrcopy (@FULL_VERSION@) unstable; urgency=low
```
Comment (drossetti) [inline packages/debian/changelog:1]: remind me why we need unstable here

**Review by drossetti — CHANGES_REQUESTED** [review]
using https://github.com/NVIDIA/gdrcopy/pull/130 as reference:
- update CHANGELOG.md
- reset DEBIAN_VERSION to 1 in build-deb-packages.sh

File: packages/debian/changelog (hunk 1 above)
Comment (pakmarkthub) [reply packages/debian/changelog]: Changed to stable. It is a legacy I forgot to change. Thanks for pointing it out.

**Comment by pakmarkthub** [comment]
@drossetti Can I ask you to take a look at the current revision?

**Review by drossetti — APPROVED** [review]

**Comment by lukeyeager** [comment]
@pakmarkthub did you mean to add a tag for this commit, too? There isn't one, currently.
https://github.com/NVIDIA/gdrcopy/tags

**Comment by pakmarkthub** [comment]
@lukeyeager Version 2.2 is not released yet. We are preparing the code for release in the near future.

**Pull request merged**